apps/web/app/(navigation)/users/page.tsx

```diff
@@ -0,0 +1,10 @@
+import { DummyAsideCardContent } from "app/DummyAsideCardContent";
+import MultiCardLayout from "../MultiCardLayout";
+
+export default function ReportsPage() {
```

Comment:
Admin-only pages should be locked behind a middleware check to not allow non-admin users access to the pages, even if they "hack" their way around the menu. Also, component name.

```suggestion
export default function UsersPage() {
```